src/libs/dotcms.api.js

```diff
@@ -44,33 +47,31 @@ const get = async ({ pathname }) => {
     const url = getUrl(pathname);
 
     return request({ url })
-        .then(data => {
+        .then((data) => {
             if (data.ok) {
                 return data.json();
             } else {
                 return data;
             }
         })
-        .then(data => {
+        .then((data) => {
             if (data.entity) {
                 return data.entity.layout ? translate(data.entity) : {};
             }
             throw new Error(data.status);
         })
-        .catch(err => {
+        .catch((err) => {
             return {
                 error: err.message
             };
         });
 };
 
-const emitNavigationEnd = pathname => {
+const emitCustomEvent = (eventName, eventData, eventType) => {
     const customEvent = window.top.document.createEvent('CustomEvent');
-    customEvent.initCustomEvent('ng-event', false, false, {
-        name: 'remote-render-edit',
-        data: {
-            pathname
-        }
+    customEvent.initCustomEvent(eventType, false, false, {
```

Comment:
The `eventType` is the same every time, you can leave the string no need to pass it everytime.

Reply:
I left it open in case we needed to send in the future other event different than `ng-event`

Reply:
No need, we listen to those events in Angular and is `ng-event` all the time.